operations/rollout-operator-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -1,10 +1,10 @@
 local utils = import 'mixin-utils/utils.libsonnet';
 
-// This is a modified version of the mimir-mixin/dashboard-utils.libsonnet
+// This is a modified version of the mimir-mixin/dashboard-utils.libsonnet.
+// Note that some fields/functions have a rop_ prefix. This has been added when the same name exists in other dashboard utils with a different signature.
```

Comment:
[nit] `rop` isn't a prefix or term we use anywhere else, so the meaning might not be immediately obvious - what if we used something like `rolloutOperator`?